src/test/java/gov/acwi/wqp/etl/BiodataBaseFlowIT.java

```diff
@@ -0,0 +1,23 @@
+package gov.acwi.wqp.etl;
+
+import org.springframework.context.annotation.Import;
+
+import com.github.springtestdbunit.annotation.DbUnitConfiguration;
+import org.junit.Before;
+import org.springframework.batch.core.JobParametersBuilder;
+
+@Import({DBTestConfig.class, BiodataDBTestConfig.class})
+@DbUnitConfiguration(databaseConnection={"wqp","pg","biodata"})
+public abstract class BiodataBaseFlowIT extends BaseFlowIT{
+	
+	@Before
+	@Override
+	public void baseSetup() {
+		testJobParameters= new JobParametersBuilder()
+				.addJobParameters(jobLauncherTestUtils.getUniqueJobParameters())
```

Comment:
The following values are now all (mostly) constants.
schemaName is EtlConstantUtils.JOB_PARM_SCHEMA - soon it's value will be set with the ConfigurationService
there is also a new EtlConstantUtils.JOB_PARM_GEO_SCHEMA also to be set via the ConfigurationService ("nwis" until then)

Reply:
Updated